webex_bot/webex_bot.py

```diff
@@ -42,6 +45,8 @@ def __init__(self,
                                       on_card_action=self.process_incoming_card_action,
                                       device_url=device_url)
 
+        self.webex_api = WebexTeamsAPI(access_token=teams_bot_token)
```

Comment:
Actually the super class has this already. Just use `self.teams` here when required.

Reply:
Thanks, done